### PR TITLE
feat: added boilr .desktop entry

### PR DESCRIPTION
**Pull request by Linwer**
GNOME users have long forgotten what a "~/Desktop" folder is, so I added a desktop entry for BoilR based on the Flatpak .desktop entry.

### CODE REVIEW

**Comment by KyleGospo** [comment]
This looks great, only request I have would be to put Boilr in ~/Applications and mkdir -p ~/Applications. We can also drop the creation of the desktop icon this way.

**Comment by Linwer** [comment]
Idk, that would mean breaking the existing workflow that KDE users (or GNOME users with the desktop icons extension) have. Having the executable in ~/Desktop doesn't and won't really bother anyone, even if inelegant.

**Comment by KyleGospo** [comment]
I'm also fine copying the .desktop file to the desktop, you can check for Kinoite before doing that as we do for others in the same file

**Comment by Linwer** [comment]
something like this then?

**Comment by KyleGospo** [comment]
Thank you!

File: system_files/deck/shared/usr/share/ublue-os/just/custom.just [hunk 1]
```diff
@@ -61,11 +61,34 @@ get-greenlight:
   chmod +x ~/Desktop/Greenlight.AppImage
 
 get-boilr:
-  wget \
+  curl \
```
Comment (szescxz) [inline system_files/deck/shared/usr/share/ublue-os/just/custom.just:64]: Missed an `-L` here.

**Pull request merged**